cask/src/cask/main/Main.scala

```diff
@@ -2,65 +2,95 @@ package cask.main
 
 import cask.endpoints.{WebsocketResult, WsHandler}
 import cask.model._
-import cask.internal.{DispatchTrie, Util}
-import cask.main
-import cask.router.{Decorator, EndpointMetadata, EntryPoint, RawDecorator, Result}
+import cask.internal.{DispatchTrie, Util, VirtualThreadBlockingHandler, VirtualThreadSupport}
+import cask.model.Response.Raw
```

Comment:
Let's avoid importing this and refer to it via the partially-qualified name `Response.Raw`